### ISSUE TITLE
Improved Input/Output mapping component is integrated in existing Input/Output tab

### ISSUE THREAD

**Issue by MaxTru** [bpmn-io/bpmn-js-properties-panel#350]
__Is your feature request related to a problem? Please describe__

We want to support the user when working with process variables. As part of this, the user shall be able to perform the input/output mapping more easily.

This feature covers the integration of the new UI component which replaces the existing input/output mapping (which is implemented as list) into the `Input/Output tab`.

__Describe the solution you'd like__
The current `Input/Output tab` uses this component:
![Screenshot_20200818_120611](https://user-images.githubusercontent.com/42800119/90501179-3c151480-e14c-11ea-9ffb-b6b57f0029e0.png)

The new component (see #349) shall be used instead. 

__Additional context__

Child of https://github.com/bpmn-io/internal-docs/issues/148
Depends on #349 
Depends on #344

**Comment by pinussilvestrus** [comment]
Maybe not in the scope of this issue (but as a next step), but here you find an overview about the usage of data input/output and camunda inputParameter/outputParameter (WIP): https://miro.com/app/board/o9J_kmFk2Kg=/

**Comment by barmac** [comment]
I just had a call with @pinussilvestrus and we clarified that the scope for this issue is to use the component developed in https://github.com/bpmn-io/bpmn-js-properties-panel/issues/349 for input/output but keep the content as it is. Then, as a follow-up issue we may implement the new content as soon as we know what we actually want to have.
